sidebars.js

```diff
@@ -170,7 +169,7 @@ module.exports = {
       "reference/cli",
       "reference/manifest",
       "reference/compose",
-      "self-hosted/administration/configuration",
+      "reference/helm-chart-settings",
```

Comment:
This was much needed, thanks! I feel having all the manifest references in one place is better idea than having the self hosted one separately!

Reply:
Glad to hear it! This was Pablo's suggestion and I think it makes sense especially as he explained the purpose of this particular page 👍